### ISSUE TITLE
Training logs of yolo models

### ISSUE THREAD

**Issue by qsunyuan** [ultralytics/ultralytics#23067]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

1. Does this repository contain logs of training YOLO models (e.g., YOLOv5s) from scratch?

Here is my code:
```python
from ultralytics import YOLO 

model = YOLO("yolov5s.yaml", task='detect')   

model.info()

results = model.train(data="coco.yaml", epochs=300, imgsz=640, batch=64, device="cuda:1")
```

2. I did not use the yolov5 repository because there might be incompatibility issues. https://github.com/ultralytics/ultralytics/issues/21914

3. In other words, I want to reproduce the same result exactly. However, the performance metric results are inconsistent. Is this due to the addition of YOLOv8 technology in the Ultralytics version?

https://github.com/ultralytics/yolov5?tab=readme-ov-file#pretrained-checkpoints

https://docs.ultralytics.com/models/yolov5/#performance-metrics

ref: https://github.com/ultralytics/yolov5/discussions/8926

**Comment by UltralyticsAssistant** [comment]
👋 Hello @qsunyuan, thank you for your interest in Ultralytics 🚀! This is an automated response to help us triage your issue quickly—an Ultralytics engineer will also assist soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report (e.g., metrics not matching expected values), please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. In addition to your snippet, an MRE should include the exact `ultralytics` version, environment details, full command/config, and the outputs/logs needed to reproduce.

If this is a custom training ❓ Question (e.g., reproducing published metrics), please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/). If you have a specific “target” result you’re trying to reproduce, please also paste the exact reference you’re matching and the metrics you’re observing.

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultrtralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
YOLOv5 in the original repo is anchor based. YOLOv5 in Ultralytics is anchor free. They are not the same. If you're asking about logs of YOLOv5 in Ultralytics, you can find it in HUB:

https://hub.ultralytics.com/models/

**Comment by qsunyuan** [comment]
thank you

**Comment by glenn-jocher** [comment]
You’re welcome — we don’t version-control “from scratch” COCO training logs in this repo, but every run writes artifacts (e.g. `results.csv`, `args.yaml`, TensorBoard events) under `runs/detect/train*` and you can view them as shown in the [Train mode docs](https://docs.ultralytics.com/modes/train/). For the closest match to the published Ultralytics YOLOv5 numbers in this package, make sure you’re comparing against the `*u` weights/recipes (e.g. `yolov5su.pt`) described in the [YOLOv5u docs](https://docs.ultralytics.com/models/yolov5/); if you still see drift, please open a new issue and attach `yolo checks` plus your run’s `results.csv`/`args.yaml`.

```bash
tensorboard --logdir runs
```
